parent/pom.xml

```diff
@@ -689,6 +689,7 @@
     <twilio-version>7.9.1</twilio-version>
     <twitter4j-bundle-version>4.0.6_1</twitter4j-bundle-version>
     <twitter4j-version>4.0.6</twitter4j-version>
+    <uberfire-maven-support-version>1.3.0.Final</uberfire-maven-support-version>
```

Comment:
optaplanner engine doesn't use uberfire, but this can't hurt either (optaplanner workbench does, but that's not used by camel)